src/scene/Scale.js

```diff
@@ -171,6 +173,13 @@ function ordinal(scale, rng, group) {
   if (!scale.invert && spatial) invertOrdinal(scale);
 }
 
+function setPadding(pad, config, defaultPad, parent_id, datum) {
+  if (!parent_id && config && config.scale && config.scale[pad] && !datum._facetID) {
+    return config.scale[pad];
+  }
+  return defaultPad;
+}
+
```

Comment:
I see, thank you! I am going to remove this for now because I think the behaviour it introduces is confusing -- two scale definitions that are otherwise identical but one at the top-level and one nested within a group would produce different visual output. More concretely, you may have a bar chart with a padded scale at the top-level, but if it were nested within a group (e.g., as part of a multi-view display), it would suddenly appear different. 

Could you open a new issue to track this for us? In particular, clearly describe what the motivation and goals are and include screenshots to illustrate examples. I wonder if there's an alternate mechanism that we can implement that provides more consistent behaviour.